### DISCUSSION TITLE
Example of a fully working blog using this package.

### DISCUSSION THREAD

**Discussion by richard457** [austintoddj/canvas#1039] [General]
Example of a fully working blog using this package. I think this can help a lot.

**Comment by austintoddj** [comment]
There's a one-click install of Canvas into the latest version of Laravel that you can check out.

In the [Contributing Guide](https://github.com/austintoddj/canvas/.github/CONTRIBUTING.md#setup), click the **Open in Gitpod** button. It's free to create an account, and you can simply sign in with GitHub.

Not only are you set up to make changes or create pull requests back to the project, but you can try out package yourself.

**Comment by austintoddj** [comment]
Closing this out with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).

The closest first-party “working blog” is the optional starter reader:

```bash
php artisan canvas:ui
```

That publishes Blade views, a controller stub, and routes (index, post, tags, topics, author, RSS) at /canvas-ui. Customize from there — Canvas is still the publishing backend inside your Laravel app, not a standalone CMS site.
